lib/WP_Auth0_LoginManager.php

```diff
@@ -156,8 +147,8 @@ public function init_auth0() {
 		// Catch any incoming errors and stop the login process.
 		// See https://auth0.com/docs/libraries/error-messages for more info.
 		if ( $this->query_vars( 'error' ) || $this->query_vars( 'error_description' ) ) {
-			$error_msg  = sanitize_text_field( $this->query_vars( 'error_description' ) );
-			$error_code = sanitize_text_field( $this->query_vars( 'error' ) );
+			$error_msg  = sanitize_text_field( rawurldecode( $_REQUEST[ 'error_description' ] ) );
```

Comment:
`$this->query_vars()` checks for WP query vars first, which was leaving out the `error` parameter for some reason. Switched to a more explicit check and now pulls `error` more consistently.

Reply:
Why does this call `rawurldecode` first when the line below doesn't?

Reply:
My thought was that the error code is usually a machine name without any characters that need decoding but that's a little bit of wishful thinking ... and no harm decoding that as well.

Reply:
Honestly, the errors returned by the server should be used directly. No need to parse nor decode them at all (besides JSON if you need to translate them to an Object structure). Some error descriptions might include URLs to help articles. I wouldn't do any urldecoding here.

Reply:
Error descriptions typically include at least spaces, if not other characters. Without urldecoding it, you'll get "This%20is%20an%20error" and such. URLs in a URL definitely need to be encoded.  

I've never seen an error code have decode-able characters, though, which was why I left it out.